test/python/providers/test_fake_backends.py

```diff
@@ -128,21 +132,25 @@ def test_circuit_on_fake_backend(self, backend, optimization_level):
             self.skipTest(
                 f"Unable to run fake_backend {backend.configuration().backend_name} without qiskit-aer"
             )
-        job = backend.run(
-            transpile(
-                self.circuit, backend, seed_transpiler=42, optimization_level=optimization_level
-            ),
-            seed_simulator=42,
-        )
-        result = job.result()
+        with warnings.catch_warnings():
+            # TODO remove this catch once Aer stops using QobjDictField
+            warnings.filterwarnings("ignore", category=DeprecationWarning, module="qiskit")
+            job = backend.run(
+                transpile(
+                    self.circuit, backend, seed_transpiler=42, optimization_level=optimization_level
+                ),
+                seed_simulator=42,
+            )
+            result = job.result()
```

Comment:
see above

Reply:
done in bd216a17e8e0f3f20b6b5862c0b7c4f20d45ff56